src/components/MegaMenu/MegaMenu.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div
+    class="block md:inline-block relative z-50 mx-0 px-0 border-b-2 border-gray-100 md:border-0 focus:border-b-2"
+    data-testId="menu-container"
+    @mouseenter="showNav = true"
+    @mouseleave="showNav = false"
+    @click="onClick"
+  >
+    <div
+      :id="dropdownToggleId"
+      class="w-full md:w-auto outline-none mx-0 py-2.5 px-3 transition-colors duration-200 ease-linear text-gray-700 whitespace-nowrap active:text-primary-300 active:no-underline hover:text-primary-300 hover:no-underline inline-block cursor-pointer relative"
+      :aria-controls="dropdownListId"
+      aria-haspopup="menu"
+    >
+      <div class="flex-nowrap flex width-100 mt-0 flex-row justify-between md:justify-start items-center">
```

Comment:
`width-100` isn't a Tailwind class so it's not doing anything here and can be removed. (unless it's supposed to be `width-full` but if it looks correct, then we can just remove)